src/test/regress/expected/multi_task_assignment_policy.out

```diff
@@ -211,4 +211,21 @@ DEBUG:  Plan is router executable
    explain statements for distributed queries are not enabled
 (2 rows)
 
+-- we should be able to use round-robin with router queries that 
+-- only contains intermediate results
+SET LOCAL citus.task_assignment_policy TO 'round-robin';
+EXPLAIN (COSTS FALSE) WITH q1 AS (SELECT * FROM task_assignment_test_table) SELECT * FROM q1;
+DEBUG:  generating subplan 14_1 for CTE q1: SELECT test_id FROM public.task_assignment_test_table
+DEBUG:  assigned task 3 to node localhost:57637
+DEBUG:  assigned task 2 to node localhost:57637
+DEBUG:  assigned task 1 to node localhost:57638
```

Comment:
is this going to be consistent if we make changes?

Reply:
I think so because the table is append distributed and the metadata is manually added.

Reply:
It seems nicer to just have a regular query somewhere.